svgnative/src/SVGDocumentImpl.cpp

```diff
@@ -703,7 +703,7 @@ void SVGDocumentImpl::ParseStrokeProperties(StrokeStyleImpl& strokeStyle, const
     {
         float miter{};
         // Miter must be bigger 1. Otherwise ignore.
-        if (SVGStringParser::ParseNumber(prop->second, miter) && miter >= 1)
+        if (SVGStringParser::ParseNumberOrPercentage(prop->second, miter) && miter >= 1)
```

Comment:
Yes i completely agree, there are no percentages for stroke-miterlimit. as i make a function generic which will works for both when you provide a number or we provide a percentage . so in this case , it will simply ignore the newly lines that added as when we have number only, in that case (pos == end) both are same.